### ISSUE TITLE
why is has parameter a object property?

### ISSUE THREAD

**Issue by wdduncan** [Buffalo-Ontology-Group/MRI_Ontology#12]
Why is `has parameter` an object property where `has MRI sequence parameter` is a data property? Perhaps it would be better to be consistent on the use of object and data properties?

**Comment by adbartni** [comment]
I agree the naming makes this pretty unclear. `has MRI sequence parameter` really deals in the realm of MRI machine setting datum values, and not necessarily with the MRI sequence parameters themselves. The subclasses of `has MRI sequence parameter` reflect this and are meant to assign the values of flip angle, TE, TR, and TI to an individual acquisition sequence, which are external values set at the MRI machine by a technician. 

I propose changing the name of `has MRI sequence parameter` to `has MRI machine setting value` and adding a definition to make this more clear.
